<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchTicketWindow</class>
 <widget class="QWidget" name="SearchTicketWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Tickets</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1301</width>
     <height>761</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>931</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font-size: 50px;
color:black;
font-weight:bold;</string>
       </property>
       <property name="text">
        <string>            Search Your Ticket Here!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_passengerName">
       <property name="styleSheet">
        <string notr="true">color:black;</string>
       </property>
       <property name="text">
        <string>Passenger Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passengerNameSearchLineEdit">
       <property name="styleSheet">
        <string notr="true">color:black;</string>
       </property>
       <property name="placeholderText">
        <string>Enter Passenger Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_bookingId">
       <property name="text">
        <string>Booking ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bookingIdSearchLineEdit">
       <property name="placeholderText">
        <string>Enter Booking ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="bookingDateSearchDateEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="ticketsTableWidget">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>20</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
